commit 9da55929ada7dccf267d64c57bfcca9792448e17
Author: Sam Thursfield <sam@afuera.me.uk>
Date:   Wed Nov 13 11:23:31 2019 +0100

    Add autogen.sh script
    
    This brings us in line with the [build
    API](https://github.com/cgwalters/build-api) used by flatpak-builder.

diff --git a/pd/src/autogen.sh b/pd/src/autogen.sh
new file mode 100755
index 00000000..3b076d1a
--- /dev/null
+++ b/pd/src/autogen.sh
@@ -0,0 +1,2 @@
+#!/bin/sh
+autoreconf -ivf
